### ISSUE TITLE
Training over-fitting after every epochs

### ISSUE THREAD

**Issue by songshan0321** [hellochick/ICNet-tensorflow#122]
I use `train.py` to train on my relabel ade20k dataset (150 to 5 classes) but couldn't get good result. mIoU is always at maximum 57%.

After plotting out the train loss and val loss graph, I found that the loss pattern is suspicious, it is shown below.
![loss](https://user-images.githubusercontent.com/21157726/69217705-3339e700-0baa-11ea-9770-a301299041dd.png)
